imageio/plugins/pillow.py

```diff
@@ -211,7 +211,13 @@ def _append_data(self, im, meta):
                 im = im[:, :, 0]
             self._written = True
             self._meta.update(meta)
-            img = ndarray_to_pil(im, self.format.plugin_id)
+            if "interpret_uint8" in self._meta:
+                img = ndarray_to_pil(
+                    im, self.format.plugin_id, self._meta["interpret_uint8"]
```

Comment:
Maybe use `self._meta.get("interpret_uint8", True)` instead of the `if-else`?

Reply:
Aye, I agree this would be cleaner. Only consequence as noted in the PR is that this shifts specification of default behaviour to `PillowFormat.Writer._append_data`. To be fair, upon further consideration this doesn't really present an issue since the plugin's code base is relatively small; I'll implement the change.